clang/test/Analysis/taint-generic.c

```diff
@@ -20,6 +22,7 @@
 // RUN: not %clang_analyze_cc1 -Wno-pointer-to-int-cast \
 // RUN:   -Wno-incompatible-library-redeclaration -verify %s \
 // RUN:   -analyzer-checker=optin.taint.GenericTaint  \
+// RUN:   -analyzer-checker=optin.taint.TaintedDiv \
```

Comment:
```suggestion
```
I strongly suspect that there is no reason to add TaintedDiv here -- this not a normal test run, but an intentionally wrong invocation that passes an invalid config file name and verifies the error output with FileCheck.

In fact, I also suspect that you could remove the ExprInspection line below this without changing the meaning of this test. (You should perhaps do it for the sake of clarity.)

Reply:
I removed the superflous options from other invocations too.